website/docs/docs/build/packages.md

```diff
@@ -157,9 +157,60 @@ packages:
 
 Where `name: 'dbt_utils'` specifies the subfolder of `dbt_packages` that's created for the package source code to be installed within.
 
-### Private packages
+## Private packages
 
```

Comment:
since we're here, maybe we can have a intro sentence here since we usually dont like having an h2 followed by an h3. so something like:

```
Private packages enable you to clone and manage private repositories in your dbt project. The following sections explain how to configure private packages using these methods:

[Native private packages](/docs/build/packages#native-private-packages) <Lifecycle status="beta"
[SSH key method (command line only)](/docs/build/packages#ssh-key-method-command-line-only)
[Git token method](/docs/build/packages#git-token-method)
```

Reply:
that way we're showing users the options for private package configurations and linking to them, while also having an intro of what private packages are.

Reply:
I'm not sure if we'll be deprecating the others, so we may not want to call attention. I'll find out and address it in a [follow up issue](https://github.com/dbt-labs/docs.getdbt.com/issues/6683)